pkg/services/alerting/notifiers/pagerduty.go

```
@@ -143,6 +158,20 @@ func (pn *PagerdutyNotifier) Notify(evalContext *alerting.EvalContext) error {
 
 	body, _ := bodyJSON.MarshalJSON()
 
+	return body, nil
+}
+
+// Notify sends an alert notification to PagerDuty
+func (pn *PagerdutyNotifier) Notify(evalContext *alerting.EvalContext) error {
+	body, err := pn.BuildEventPayload(evalContext)
+	if err != nil {
+		pn.log.Error("Unable to build event pagerduty payload: ", err)
```

Please return the error here.

---

ack